### ISSUE TITLE
Studio : examples screwed after update

### ISSUE THREAD

**Issue by john-doe-2016** [stoplightio/studio#251]
After a recent update, all my objects' samples are screwed.
Apparently, you removed the 'value' attribute that was introducing samples data, but did not clean up after the update.

Thus, all my samples now have a 'value' root object.
For example
```javascript
{
  "value": {
    "persTitleId": 18,
    "persGenderCode": "M",
    "translatedLabels": [
      {
        "languageCode": "fr",
        "translation": "M"
      },
      {
        "languageCode": "en",
        "translation": "Mr"
      },
      {
        "languageCode": "es",
        "translation": "Sr"
      }
    ]
  }
}
```

Instead of
```javascript
{
  "persTitleId": 18,
  "persGenderCode": "M",
  "translatedLabels": [
    {
      "languageCode": "fr",
      "translation": "M"
    },
    {
      "languageCode": "en",
      "translation": "Mr"
    },
    {
      "languageCode": "es",
      "translation": "Sr"
    }
  ]
}
```

Can you confirm that this 'value' removal is permanent and not the result of a temporary bug?
I do not want to update all my samples (50+) if this 'value' removal is not your design.

Regards

**Comment by lottamus** [comment]
@john-doe-2016 yes we're very sorry about that! That bug was reported in [this issue](https://github.com/stoplightio/studio/issues/176) and was fixed in [v1.5.0](https://support.stoplight.io/hc/en-us/articles/360035807092). 

You're safe to delete the "value" property from examples in any standalone JSON Schema models. The extra "value" property should only be added for [examples in OpenAPI v3](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#example-object).

**Comment by john-doe-2016** [comment]
Hi @lottamus ,

OK, thanks for the information. Samples corrected :)
